gradient/commands/notebooks.py

```diff
@@ -249,3 +207,24 @@ def _make_table(table_data):
         ascii_table = terminaltables.AsciiTable(table_data)
         table_string = ascii_table.table
         return table_string
+
+
+class NotebookLogsCommand(LogsCommandMixin, BaseNotebookCommand):
+    def _make_table(self, logs, id):
+        table_title = "Notebook %s logs" % id
+        table_data = [("LINE", "MESSAGE")]
+        table = terminaltables.AsciiTable(table_data, title=table_title)
+
+        for log in logs:
+            table_data.append(self._format_row(log))
+
+        return table.table
+
+    def _get_log_row_string(self, id, log):
+        log_msg = "{}\t{}".format(*self._format_row(log))
+        return log_msg
+
+    @staticmethod
+    def _format_row(log_row):
+        return (style(fg="red", text=str(log_row.line)),
+                log_row.message)
```

Comment:
Why is entire class rewritten here and not in place? This makes no sense and only makes it harder to follow changes

Reply:
Implementations are now consolidated.